cf/dimensioncoordinate.py

```diff
@@ -339,155 +339,223 @@ def create_bounds(
-                        For coordinates ``1984-12-01, 1984-12-02,
-                        2000-04-15`` setting, ``cellsize=cf.D(hour=12)``
-                        will result in bounds of ``(1984-11:30 12:00,
-                        1984-12-01 12:00), (1984-12-01 12:00, 1984-12-02
-                        12:00), (2000-05-14 12:00, 2000-04-15 12:00)`` (see
-                        `cf.D` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-12-16 12:00, 1985-01-16
-                        12:00`` setting, ``cellsize=cf.M()`` will result in
-                        bounds of ``(1984-12-01, 1985-01-01), (1985-01-01,
-                        1985-02-01)`` (see `cf.M` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-12-01 12:00, 1985-01-01
-                        12:00`` setting, ``cellsize=cf.M()`` will result in
-                        bounds of ``(1984-12-01, 1985-01-01), (1985-01-01,
-                        1985-02-01)`` (see `cf.M` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-12-01 12:00, 1985-01-01
-                        12:00`` setting, ``cellsize=cf.M(day=20)`` will
-                        result in bounds of ``(1984-11-20, 1984-12-20),
-                        (1984-12-20, 1985-01-20)`` (see `cf.M` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-03-01, 1984-06-01`` setting,
-                        ``cellsize=cf.Y()`` will result in bounds of
-                        ``(1984-01-01, 1985-01-01), (1984-01-01,
-                        1985-01-01)`` (see `cf.Y` for details). Note that in
-                        this case each cell has the same bounds. This
-                        because ``cf.Y()`` is equivalent to ``cf.Y(month=1,
-                        day=1)`` and the closest 1st January to both
-                        coordinates is 1st January 1984.
-
-                {+data-like-scalar} TODO
-
-            flt: `float`, optional
-                When creating cells with sizes specified by the *cellsize*
-                parameter, define the fraction of the each cell which is
-                less its coordinate value. By default *flt* is 0.5, so that
-                each cell has its coordinate at it's centre. Ignored if
-                *cellsize* is not set.
+                *Parameter example:*
+                  For coordinates ``1, 2, 10``, setting ``bound=0.5``
+                  will result in bounds of ``(0.5, 1.5), (1.5, 2.5),
+                  (2.5, 17.5)``.
 
                 *Parameter example:*
-                    For coordinates ``1, 2, 10``, setting ``cellsize=1,
-                    flt=0.5`` will result in bounds of ``(0.5, 1.5), (1.5,
-                    2.5), (9.5, 10.5)``.
+                  Beware that if *bound* is too large (small), then
```

Comment:
```suggestion
                  Beware that if *bound* is too large or small, then
```

since in this case there is no matched component in parentheses later in the sentence, so it is somewhat confusing?